### DISCUSSION TITLE
how to keep orientation of pictures

### DISCUSSION THREAD

**Discussion by dirkpauli** [gerbera/gerbera#2336] [Q&A]
When importing (jpg) pictures into Gerbera, it seems to ignore the orientation of the picture that is set in the exif tag. Portrait images are rotated as landscape in both the thumbnail as well as when displaying it on the client.

I already added the EXIF_TAG_ORIENTATION tag to the library options of libexif, but this changes nothing.
Any advise?

**Comment by whyman** [comment]
This likely will be an issue with your renderer not respecting the exif.

To work around it you may need to set up a transcoding profile to generate a new pre-rotated image for the renderer.

**Reply by KarlStraussberger** [reply]
It is correct that the web ui does not respect the orientation neither for the thumb nor for the image. Best to rotate images before importing them - ms windows explorer has the same issue.

**Comment by dirkpauli** [comment]
Thanks for that hint, Ian! Is there anywhere a documention how to set up such a transcoding profile?

 
 

Gesendet: Freitag, 17. Dezember 2021 um 10:54 Uhr
Von: "Ian Whyman" ***@***.***>
An: "gerbera/gerbera" ***@***.***>
Cc: "dirkpauli" ***@***.***>, "Author" ***@***.***>
Betreff: Re: [gerbera/gerbera] how to keep orientation of pictures (Discussion #2336)


 

This likely will be an issue with your renderer not respecting the exif.

To work around it you may need to set up a transcoding profile to generate a new pre-rotated image for the renderer.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by KarlStraussberger** [reply]
There is a discription https://docs.gerbera.io/en/latest/transcoding.html

And you can also check https://docs.gerbera.io/en/latest/config-transcode.html

**Comment by Maikl963** [comment]
Has anyone managed to fix the orientation of the images?

I have spent several hours trying but to no avail, thanks a lot for the advice.
